conf/messages

```diff
@@ -144,5 +144,6 @@ transferSummary.progress=5
 progress.totalSteps=5
 
 signedOut.header=Signed out
-signedOut.title=You have signed out
+signedOut.title=You have successfully signed out
+signedOut.paragraph=Thanks for using the Transfer Digital Records service.
```

Comment:
It would be good to give this a more descriptive name than "paragraph" so its purpose is clearer in the template. What about `signedOut.thankYouMessage`?

Reply:
Sounds good!